src/web/services/sessions.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { OngoingSessions } from '../types/api-output';
+import { HttpRequestService } from './http-request.service';
+
+/**
+ * Handles sessions related logic provision.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class SessionsService {
```

Comment:
Call it as `FeedbackSessionsService` cause `Sessions` is ambiguous in HTTP context.